### ISSUE TITLE
Style/TrailingUnderscoreVariable cop leaves an unclosed parenthesis

### ISSUE THREAD

**Issue by guih** [bbatsov/rubocop#4563]
`Style/TrailingUnderscoreVariable` cop wrongly fixes
```ruby
(foo, _) = [1, 2, 3]
```
into
```ruby
(foo, = [1, 2, 3]
```
and breaks the syntax.

Maybe the problem is not with this specific cop and the parenthesis should be removed first by another cop.

Otherwise, it should not remove the closing parenthesis.

--------

## Expected behavior

It should replace to `foo, = [1, 2, 3]` or `(foo, ) = [1, 2, 3]`

## Actual behavior

It leaves an unclosed parenthesis and breaks the syntax.

## Steps to reproduce the problem

```ruby
# test.rb
(foo, _) = [1, 2, 3]
puts foo
```

```shell
rubocop -a test.rb
```

## RuboCop version

```
$ rubocop -V
0.49.1 (using Parser 2.3.3.1, running on ruby 2.3.3 x86_64-darwin15)
```

**Comment by Drenmi** [comment]
> Maybe the problem is not with this specific cop and the parenthesis should be removed first by another cop.

There might be room for another cop to remove the parentheses, but we can't rely on that being enabled, so you're correct in saying this cop should not break the code. 🙂
